src/app/icd/ICDManager.cpp

```diff
@@ -52,6 +52,12 @@ void ICDManager::Init(PersistentStorageDelegate * storage, FabricTable * fabricT
     VerifyOrDie(stateObserver != nullptr);
     VerifyOrDie(symmetricKeystore != nullptr);
 
+    bool supportLIT = SupportsFeature(Feature::kLongIdleTimeSupport);
+    VerifyOrDieWithMsg((supportLIT == false) || (supportLIT && SupportsFeature(Feature::kCheckInProtocolSupport)), AppServer,
```

Comment:
```suggestion
    VerifyOrDieWithMsg((supportLIT == false) || SupportsFeature(Feature::kCheckInProtocolSupport), AppServer,
```
From the boolean standpoint, you don't need to check if supportLit == True since you know it is true if you are checking for the Check-In Protocol Support.